dashboard/test/ui/features/instructions/csp_top_instructions_eyes.feature

```diff
@@ -8,12 +8,20 @@ Scenario: CSD and CSP Top Instructions
   And I wait for the page to fully load
   Then I see no difference for "teacher in applab level with rubric"
   Then I click selector ".uitest-feedback"
+  And I wait for the page to fully load
   Then I see no difference for "teacher in applab level viewing rubric"
   Then I click selector ".uitest-instructionsTab"
+  And I wait for the page to fully load
   Then I see no difference for "teacher in applab level with rubric after viewing rubric"
 
   And I am on "http://studio.code.org/s/allthethings/stage/38/puzzle/2?enableExperiments=2019-mini-rubric"
-  And I wait to see ".uitest-feedback"
+  And I wait until element ".user_menu" is visible
+  And I wait until element "iframe" is visible
+  And I switch to the first iframe
+  And I wait to see "#bramble"
+  And I switch to the first iframe
```

Comment:
to keep this test from being flaky, I think you may want to explicitly wait for this iframe to come into view before switching to it. I think you can do this by adding `And I wait until element "iframe" is visible` just before this line.